opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java

```diff
@@ -649,7 +654,7 @@ private void rebuildTagList(File directory) {
             // In case of partial success, do not null-out tagList here.
         }
 
-        if (LOGGER.isLoggable(Level.FINEST)) {
+        if (LOGGER.isLoggable(Level.FINER)) {
```

Comment:
Why this change? It does not make much sense to check for higher log level, right?

Reply:
I think it should be higher than the `FINEST` log above.